testsuite/gnat2goto/README.md

```diff
@@ -89,6 +89,24 @@ Requirements
 
 Running tests
 -------------
+
+Ensure that both gnat2goto and cbmc are on the path:
+
+```
+export PATH=${PATH}:/path/to/gnat2goto/lib/cbmc/build/bin:/path/to/gnat2goto/gnat2goto/install
```

Comment:
That example path for CBMC is only valid if you built with cmake and specifed `build` as the build directory. If you built CBMC using make, it'd be something like `/path/to/gnat2goto/lib/cbmc/src/cbmc` I think?

Reply:
Yes it is and please can we have that version because not all upstream users use cmake.

Reply:
Can do - though not all upstream users use make either (points at self). I figured the above was clear that it refers to wher ever the build is (whereas pointing at `src` you might wrongly think it needs to be the src tree). 

Will amend to:

> ```
> export PATH=${PATH}:/path/to/gnat2goto/lib/cbmc/path/to/cbmc:/path/to/gnat2goto/gnat2goto/install
> ```
>
> Note for `make` users `/path/to/cbmc` will point to `lib/cbmc/src/cbmc`. For `cmake` it will be something like `lib/cbmc/build/bin`.